dolomite-uniform/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 version: '3'
 services:
-  rock-mediation:
+  rock-dolomite-uniform:
     image: datashield/rock-dolomite-uniform:latest
```

Comment:
You have changed 'latest' for '1.0.1', make sure this compose file is consistent then, or revert tag default value to 'latest' (version tag can be specified on the command line like `make build-image no_cache=true tag=1.0.1`).

Reply:
I going to make "1.0.1" into "latest" after it has been tested.

Reply:
What I do is:

```
make build-image no_cache=true tag=1.0.1
```

and later:

```
make build-image tag=latest
```

The idea is to minimize the required changes for each version in the project files.

Reply:
`latest` means latest release, and usually it is not recommended to use this tag as you do not know what you get...

Reply:
I think this should be added to the list of things to be document.